### DISCUSSION TITLE
Transparent Mode with upstream Proxy Chaining?

### DISCUSSION THREAD

**Discussion by shockogit** [mitmproxy/mitmproxy#6407] [Q&A]
I have a specific use case to transparently intercept all HTTP/HTTPS requests (working via `--mode transparent`) but then chain them to an upstream proxy. Is this possible with MITM?

**Comment by mhils** [comment]
That is currently not supported out of the box. You may be able to do that with a custom addon.

**Comment by malwareinfosec** [comment]
@mhils bumping this thread, is chaining ``transparent`` with ``upstream`` supported?
